htlcswitch/switch.go

```diff
@@ -67,6 +67,10 @@ var (
 	// ErrUnreadableFailureMessage is returned when the failure message
 	// cannot be decrypted.
 	ErrUnreadableFailureMessage = errors.New("unreadable failure message")
+
+	// ErrLocalAddFailed signals that the ADD htlc for a local payment
+	// failed to be processed.
+	ErrLocalAddFailed = errors.New("local add HTLC failed")
```

Comment:
Does this need to be exported?

Reply:
Not strictly, but this error is returned to other sub-systems. By exporting it, those could check for it without string matching.